exporters/otlp/src/OpenTelemetry/Exporter/OTLP/Span.hs

```diff
@@ -108,9 +109,9 @@ data OTLPExporterConfig = OTLPExporterConfig
   { otlpEndpoint :: Maybe String
   , otlpTracesEndpoint :: Maybe String
   , otlpMetricsEndpoint :: Maybe String
-  , otlpInsecure :: Maybe Bool
-  , otlpSpanInsecure :: Maybe Bool
-  , otlpMetricInsecure :: Maybe Bool
+  , otlpInsecure :: Bool
+  , otlpSpanInsecure :: Bool
+  , otlpMetricInsecure :: Bool
```

Comment:
Surely these are used by something that'll break now, right? :thinking:

Reply:
CI passes, so I guess not. It seems they're just parsed and not used.